ray-operator/controllers/ray/utils/constant.go

```diff
@@ -188,6 +188,11 @@ const (
 	// as well as the user-agent. This constant is updated before release.
 	// TODO: Update KUBERAY_VERSION to be a build-time variable.
 	KUBERAY_VERSION = "nightly"
+
+	// KubeRayController represents the value of the default job controller
+	KubeRayController = "ray.io/kuberay-operator"
+	// MultiKueueController represents the vaue of the MultiKueue controller
+	MultiKueueController = "kueue.x-k8s.io/multikueue"
```

Comment:
Is this const actually used in the controller? If it's only used in tests I suggest moving it to the test file

Reply:
At least KubeRayController is used in the `rayjoob_controller.go`

Reply:
yes, I was specifically referring to MultiKueueController

Reply:
got it